tokio-console/src/state/tasks.rs

```diff
@@ -354,6 +354,10 @@ impl Task {
         self.stats.last_wake > self.stats.last_poll_started
     }
 
+    pub(crate) fn is_blocking(&self) -> bool {
+        self.kind.as_ref() == "block_on" || self.kind.as_ref() == "blocking"
```

Comment:
Take it or leave it: this could also be written using pattern matching, using the [`matches!` macro](https://doc.rust-lang.org/stable/std/macro.matches.html). So, you could also write:
```suggestion
        matches!(self.kind.as_ref() , "block_on" | "blocking")
```
which is maybe a bit more concise.

I'm fine with either version of this code, though!